spec/ic.did

```diff
@@ -49,8 +49,7 @@ service ic : {
       module_hash: opt blob;
       memory_size: nat;
       cycles: nat;
-      freezing_threshold: nat;
-      idle_cycles_burned_per_second: float64;
+      idle_cycles_burned_per_hour: nat;
```

Comment:
Is `cycles per hour` sufficient for the increase in precision? I was thinking maybe to use ` per day`.

Reply:
It seems that based on 5 Tc/GB/Year, I get to 5000 c/Byte/year, or around 0.5 c/Byte/hour. Which would mean that by specifying the price per hour we indeed give up on some precision. I don't think it's a big deal either way, but based on the above argument I have a slight preference for the day option.